### PR TITLE
Fix unchecked errors in clusterloader2 code.

### PR DESCRIPTION
**Pull request by oxddr**
Those were caught by errcheck.

/assign @krzysied

### CODE REVIEW

**Comment by wojtek-t** [comment]
LGTM once compile errors are fixed

File: clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects.go [hunk 1]
```diff
@@ -200,7 +200,10 @@ func getSelectorFromUnstrutured(obj *unstructured.Unstructured) (labels.Selector
 			return nil, fmt.Errorf("try to selector failed, field selector not found")
 		}
 		var selector metav1.LabelSelector
-		runtime.DefaultUnstructuredConverter.FromUnstructured(selectorMap, &selector)
+		err = runtime.DefaultUnstructuredConverter.FromUnstructured(selectorMap, &selector)
+		if err != nil {
+			return nil, fmt.Errorf("trye to selector failes, %v", err)
```
Comment (krzysied) [inline clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects.go]: nit: typo

**Comment by oxddr** [comment]
PTAL - tests are passing now.

**Pull request merged**